### DISCUSSION TITLE
Working with trustzone

### DISCUSSION THREAD

**Discussion by Farossco** [zephyrproject-rtos/zephyr#36128] [Q&A] [answered]
Hi, I've recently been diving into the whole trustzone implementation on Zephyr and I'm having trouble understanding how to properly work with it.

My question is fairly simple: is there a way to develop two separate pieces of code (a secure and a non-secure) both using zephyr and have them built and merged together in a simple manner, using west for example ?

**Comment by torntrousers** [comment, marked as answer]
As a nudge - I'm interested in any answers about this too

**Reply by Farossco** [reply]
Hi @torntrousers,

I got an answer form Nordic's DevZone: https://devzone.nordicsemi.com/f/nordic-q-a/76241/working-with-trustzone
